packages/charm/src/iframe/prompt.ts

```diff
@@ -1,18 +1,16 @@
 import { JSONSchema } from "@commontools/builder";
 import {
-  DEFAULT_MODEL_NAME,
+  applyDefaults,
+  type GenerationOptions,
   hydratePrompt,
   type LLMMessage,
   type LLMRequest,
-  applyDefaults,
-  type GenerationOptions
 } from "@commontools/llm";
-
-import { extractUserCode, systemMd } from "./static.ts";
+import { extractUserCode, staticSystemMd } from "./static.ts";
 
 export const RESPONSE_PREFILL = "```javascript\n";
 
-export const buildPrompt = ({
+export const buildPrompt = async ({
```

Comment:
Using lazy-loaded prompts incurs an async barrier here -- I wanted to use a prompt that's used in the integration test, and making `buildPrompt` async was the easiest way forward, no opinions here on how prompts should be handled. Can update to whatever preferred interface you'd like here for this prompt @bfollington, but want to limit to porting a single prompt for now

Reply:
This doesn't seem like a big deal to me, we're already awaiting basically every second line during the generation flow as it stands. We can move towards this lazy import pattern for prompts as we migrate each one individually.